examples/ct/handlers.go

```diff
@@ -86,6 +89,10 @@ func (a appHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	status, err := a.handler(ctx, a.context, w, r)
 	glog.V(2).Infof("%s: %s <= status=%d", a.context.logPrefix, a.name, status)
+	a.context.expAllRsps.Add(strconv.Itoa(status), 1)
+	if e := a.context.expRsps.Get(a.name).(*expvar.Map); e != nil {
```

Comment:
Maybe do `e, ok := ...(*expvar.Map)` and check ok otherwise it'll panic if what you get somehow isn't an `*expvar.Map`.